packages/x-date-pickers/src/DateField/DateField.types.ts

```diff
@@ -53,12 +53,13 @@ export type DateFieldOwnerState<TDate> = DateFieldProps<TDate>;
 
 export interface DateFieldSlotsComponent {
   /**
-   * Input rendered.
-   * @default TextField
+   * Form control with an input to render the value.
+   * Receives the same props as '@mui/material/TextField'.
```

Comment:
```suggestion
   * Receives the same props as `@mui/material/TextField`.
```
Same suggestion would apply to all components having this JSDoc.
![Screenshot 2023-01-03 at 14 03 55](https://user-images.githubusercontent.com/4941090/210353966-8b27c110-2483-420e-8d87-83d01d78fa8d.png)

Reply:
Done :+1: